models/models.js

```diff
@@ -8,6 +8,20 @@ exports.selectAllTopics = () => {
     })
 }
 
+exports.selectArticleById = (artId) => {
+    return db.query(`SELECT * FROM articles WHERE article_id = $1`, [artId])
+    .then(({ rows }) => {
+        const article = rows
+        if (article.length === 0) {
+            return Promise.reject({
+                status: 404,
+                msg: `article does not exist`
+            })
+        }
+        return article
```

Comment:
should ideally send back the object here not the array, just so that you don't need to do so in the controller